docs/api/ctx.md

```diff
@@ -1442,6 +1442,24 @@ Redirects to the URL derived from the specified path, with specified status, a p
 
 :::info
 If **not** specified, status defaults to **302 Found**.
+
+In Fiber v3, the default will change to **303 See Fount** for more predictable
```

Comment:
Just remove any comments about v3 here, since these docs update is v2 related

Reply:
@andradei

Reply:
Done